contracts/hardhat.config.ts

```diff
@@ -1,8 +1,46 @@
+require('dotenv').config({ path: __dirname + '/.env' })
 import { HardhatUserConfig } from "hardhat/config";
 import "@nomicfoundation/hardhat-toolbox";
 
+const PVT_KEY: string = process.env.PVT_KEY ?? ""
+const ETHERSCAN_API_KEY: string = process.env.ETHERSCAN_API ?? ""
+const ARBISCAN_API_KEY: string = process.env.ARBISCAN_API ?? ""
+
 const config: HardhatUserConfig = {
   solidity: "0.8.21",
+  networks: {
+    hardhat: {
+    },
+    sepolia: {
+      url: "https://eth-sepolia.g.alchemy.com/v2/CvGnxp1M23ddg9DSf9ZwNSIXXqPzS52E",
+      accounts: [PVT_KEY]
+    },
+    goerli: {
+      url: "https://eth-goerli.g.alchemy.com/v2/CvGnxp1M23ddg9DSf9ZwNSIXXqPzS52E",
+      accounts: [PVT_KEY],
+    },
+    arbisepolia: {
+      url: "https://arb-sepolia.g.alchemy.com/v2/mjXgVMd5C1IQaZYRMOv_tCC6GqwPMRkx",
+      accounts: [PVT_KEY]
+    },
```

Comment:
We should deploy on Arbitrum-Goerli instead of Arbitrum-Sepolia since the Savvy contract exists in the former

Reply:
okay will update this.
Please send some arbi gorli eth to this address ```0x4A1F47a15831A5f4Cf627414BF57145B0b47de1a```
I can't get any of the faucets to work